_shows/17_18/this_changes_everything.md

```diff
@@ -0,0 +1,104 @@
+---
+title: This Changes Everything
+playwright: Joel Horwood
+season: Postgrads
+season_sort: 160
+period: Spring
+venue: New Theatre
+date_start: 2018-01-24
+date_end: 2018-01-27
+
+cast:
+- role: Angus
+  name: Will Willmane
+  person: false
+- role: Ali
+  name: Ola Czerwiakowska
+  person: false
+- role: Alva
+  name: Laura De Simoni
+  person: false
+- role: Basic Jayne
+  name: Fabio Gentile
+  person: false
+- role: Ebba
+  name: Sharmilla Sunsurooah
+  person: false
+- role: Elin
+  name: Julie Liao
+  person: false
+- role: Eve and/or Evie
+  name: Julie Daujat
+  person: false
+- role: Freya
+  name: Lola Vazquez Peraita
+  person: false
+- role: Henri
+  name: Carmine Napoli
+  person: false
+- role: Kim
+  name: Judy Zhao
+  person: false
+- role: Maja
+  name: Adele Nasti
+  person: false
+- role: Malin
+  name: Carla Martin Clave
+  person: false
+- role: Moa
+  name: Cate Zerega
+  person: false
+- role: Sam
+  name: Pete Iwanciw
+  person: false
+- role: Tuva
+  name: Pietro Luizzo Scorpo
+  person: false
+- role: Klara
+  name: Ulli Kuchner
+  person: false
+- role: Hanne
+  name: Esther Camacho Cerezo
+  person: false
+- role: Lucy
+  name: Mariele Valci
+  person: false
+- role: Lee (Lisa)
+  name: Andrés Vallellano
+  person: false 
+
+crew:
+- role: Producer
+  name: Julie Daujat
+  person: false
+- role: Producer
+  name: Chris Ward
+  person: false
+- role: Lighting Designer
+  name: Sam Osborne
+- role: Technical Director
+  name: Beatrice Wallbank
```

Comment:
Is this an NT or Postgrad person?

Reply:
She is a postgrad but is joining nnt as actually really good